### PR TITLE
chore(NcEllipsisedOption): add docs

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

- For: https://github.com/nextcloud-libraries/nextcloud-vue/issues/5662
- cc @DaphneMuller

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

### CODE REVIEW

File: src/components/NcEllipsisedOption/NcEllipsisedOption.vue [hunk 1]
```diff
@@ -20,6 +20,54 @@
...
 
+<docs>
+A component to display a long text with highlight support in one line truncated with ellipsis in the end but keeping up to 10 last characters.
+
+It is supposed to be used as an `NcSelect`'s option in first place.
+
+```vue
+<template>
+	<div>
+		<h5>Plain text</h5>
```
Comment (susnux) [inline src/components/NcEllipsisedOption/NcEllipsisedOption.vue:31]: should be h4 no?

Comment (ShGKme) [reply src/components/NcEllipsisedOption/NcEllipsisedOption.vue:31]: Looks too huge to me 👀

But can replace it with `h4`

Comment (ShGKme) [reply src/components/NcEllipsisedOption/NcEllipsisedOption.vue:31]: Or with separated examples

Comment (ShGKme) [reply src/components/NcEllipsisedOption/NcEllipsisedOption.vue:31]: I'll do it

Comment (ShGKme) [reply src/components/NcEllipsisedOption/NcEllipsisedOption.vue]: Done

**Review by susnux — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Pull request merged**